web/src/reducers/index.js

```diff
@@ -26,7 +26,7 @@ const initialApplicationState = {
   co2ColorbarValue: null,
   colorBlindModeEnabled: cookieGetBool('colorBlindModeEnabled', false),
```

Comment:
is it only wind/solar where the cookie is not set/restored?

Reply:
Yes, the reason being that `solar` and `wind` are search params reflected in the URL state while the other 3 bools for which we still use cookies (`colorBlindModeEnabled`, `brightModeEnabled`, `onboardingSeen`) currently exist _only_ in Redux. Two points though:

* If we really want to keep the cookies for `solar` and `wind`, I could re-add them - the mechanism would just have to be a bit different as we'd have to redirect the URL to reflect the cookie values and update the cookies on URL change
* IMO we should also move some of the 3 aformentioned bools from Redux to the URL state -as I suggested in #2314